python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py

```diff
@@ -0,0 +1,683 @@
+import pytest
+import os
+import shutil
+import random
+import subprocess
+import configsuite
+import numpy as np
+import pandas as pd
+
+from copy import deepcopy
+from collections import namedtuple
+
+from res.enkf import ResConfig, EnKFMain, ConfigKeys
+from res.enkf.jobs.scaling import (scaling_job, job_config,
+                                   measured_data, scaled_matrix)
+from ecl.util.util import BoolVector
+from res.enkf import ErtRunContext
+
+_TEST_DATA_DIR = os.path.abspath(os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), os.path.join("..", "..", "..", "..", "..", "test-data")
```

Comment:
`os.path.join("..", "..", "..", "..", "..", "test-data")` might as well be `"../../../../../test-data"`. De facto separator in libres is `/`.

Reply:
Don't quite agree, shouldn't we strive for cross-platform compatibility if we can?

Reply:
You are correct, but I thought the set of platforms were all unix/linux, of which all use `/` as path separator.